intrinsics_tester/intrinsics-tester.cpp

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2021-present The Alive2 Authors.
+// Distributed under the MIT license that can be found in the LICENSE file.
+// Version: February 23, 2022
+
+#include "JIT.h"
+#include "commandLineUtil.h"
+#include "compareFunctions.cpp"
+#include "irGenerator.h"
+#include "irWrapper.h"
+#include "progressBar.h"
+#include "randomizer.h"
+#include "testLoop.h"
+#include "vectorWrapper.h"
+#include "vectorutil.h"
+#include "x86Intrin.h"
+#include <chrono>
+
+int main(int argc, char **argv) {
+  CommandLineUtil commandLineUtil(argc, argv);
+
+  bool enableDebug = CommandLineUtil::useDebugMode;
+  unsigned lowerBound = CommandLineUtil::lowerInclusiveBound;
+  unsigned upperBound = CommandLineUtil::upperExclusiveBound;
+  // Initialize llvm module and intrinsic module
+  InitializeModule();
+
+  std::unique_ptr<llvm::orc::JIT> JITCompiler = Tester::generateIntrinsicJIT();
```

Comment:
For alive2's convention, for .cpp files we prefer omit the namespace for types and put "use namespace ...;" at the top of the file.